dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1494,7 +1494,7 @@ def script_level_params(script_level)
     assert_response :success
 
     hidden = JSON.parse(response.body)
-    assert_equal [@custom_lesson_1.id.to_s], hidden
+    assert_equal [@custom_lesson_1.id], hidden
```

Comment:
The shape returned from this API for students is now technically different - It used to be `["123"]` and now it's `[123]` - but I manually tested to confirm that it still works with the javascript code. It doesn't make a ton of sense why they were returned as strings to begin with.